setup.py

```diff
@@ -65,7 +65,7 @@ def get_git_sha():
     packages=find_packages(),
     include_package_data=True,
     zip_safe=False,
-    scripts=["superset/bin/superset"],
+    entry_points={"console_scripts": ["superset=superset.cli:superset"]},
```

Comment:
`mypy` via `pre-commit` had a conflict between `superset/__init__.py` and this script and thus I opted for an alternative formulation of the script definiton.